RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ConfirmSkippingBDFS/ConfirmSkippingBDFS+View.swift

```diff
@@ -0,0 +1,62 @@
+// MARK: - ConfirmSkippingBDFS.View
+extension ConfirmSkippingBDFS {
+	@MainActor
+	public struct View: SwiftUI.View {
+		private let store: StoreOf<ConfirmSkippingBDFS>
+
+		public init(store: StoreOf<ConfirmSkippingBDFS>) {
+			self.store = store
+		}
+
+		public var body: some SwiftUI.View {
+			VStack(spacing: .medium2) {
+				// FIXME: Strings
+				Text("No Main Seed Phrase?")
+					.textStyle(.sheetTitle)
+					.padding(.horizontal, -.small2)
+
+				Text("WARNING: If you continue without entering your previous main “Babylon” seed phrase, **you will permanently lose access** to your Personas and any Accounts listed on the previous screen. A new main seed phrase will be created.\n\nTap Continue to proceed with recovering control of any Accounts created with a Ledger hardware wallet, or Accounts you originally created on the Olympia network.")
```

Comment:
see question about copy for Matt: https://rdxworks.slack.com/archives/C031A0V1A1W/p1700054688955569?thread_ts=1700051937.477939&cid=C031A0V1A1W

Reply:
Copy updated